WcaOnRails/app/webpacker/javascript/results/NewResult/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import { registerComponent } from '../../wca/react-utils';
+import ResultForm from '../ResultForm/ResultForm';
+
+const NewResult = ({
+  result,
+}) => (
+  <>
+    <h3>Creating a new result</h3>
+    <ResultForm result={result} sync={() => {}} />
```

Comment:
@saranshgrover this is the entry point for the "new result" component, here we feed the generic result form with an almost empty result, and we pass a noop for the sync method, since there is nothing to sync.